src/components/Card/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Statistic from "../Statistic/Statistic"
+
+class Card extends React.Component {
```

Comment:
This component also doesn't use state and so could be transformed into a stateless component

Reply:
on it